src/Collection.php

```diff
@@ -72,16 +72,24 @@ public function endpoint_callback( \WP_REST_Request $request ) {
 	protected function loop() {
 		$data = [];
 
+		$meta = [];
+
 		$this->query = new \WP_Query( $this->args );
 		while ( $this->query->have_posts() ) {
 			$this->query->the_post();
 			$data[] = $this->format_item( $this->query->post );
+
+			if ( empty( $meta ) ) {
```

Comment:
Good question... it's because that's what Yoast does. I just copied the same functionality. At some point we could add a way to override it, but for now I think it's good enough if it's what Yoast is doing.